config.env

```diff
@@ -3,6 +3,8 @@
 MODULE_NAME=http-ingress
 MODULE_TYPE=INGRESS
 EGRESS_URL=https://hookb.in/YVDgXMdkmbsQERGGEQ99
-
+AUTHENTICATION_REQUIRED=no
+ACCESS_TOKEN=None
+AUTHENTICATION_API_KEY=None
```

Comment:
We don't need this changes. Please, revert it.

Reply:
done.